test/app-tap/datetime.test.lua

```diff
@@ -375,6 +379,95 @@ test:test("Formatting limits", function(test)
     end
 end)
 
+test:test("Simple tests for parser", function(test)
+    test:plan(4)
+    test:ok(date.parse("1970-01-01T01:00:00Z") ==
+            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0})
+    test:ok(date.parse("1970-01-01T02:00:00+02:00") ==
+            date.new{year=1970, mon=1, day=1, hour=2, min=0, sec=0, tzoffset=120})
+
+    test:ok(date.parse("1970-01-01T02:00:00Z") <
+            date.new{year=1970, mon=1, day=1, hour=2, min=0, sec=1})
+    test:ok(date.parse("1970-01-01T02:00:00Z") <=
+            date.new{year=1970, mon=1, day=1, hour=2, min=0, sec=0})
+end)
+
+test:test("Multiple tests for parser (with nanoseconds)", function(test)
+    test:plan(193)
```

Comment:
Seems testcases for rfc3339 are absent, right?
RFC contains 4 examples, see https://datatracker.ietf.org/doc/html/rfc3339#section-5.8.

Reply:
Well, rfc3339 is particular profile for iso-8601, and examples mentioned in the rfc are valid iso8601 strings and should be covered in general. Though I'll make sure we have mentioned specifically all formats permutations. Thanks!